okio-files/src/commonTest/kotlin/okio/files/FileSystemTest.kt

```diff
@@ -29,12 +29,16 @@ import kotlin.test.assertFailsWith
 import kotlin.test.assertFalse
 import kotlin.test.assertTrue
 
+expect fun tmpDirectory(): String
```

Comment:
Should we put this in `Filesystem` ?

Reply:
Done. There's a nice rabbit hole of naming/API designs questions though...
* `tmpDirectory` vs `tempDirectory`
* should it return a `Path` instead of String?
* maybe expose the `prefix` paramater?